src/components/Nav.jsx

```diff
@@ -1,13 +1,18 @@
 import { Link } from "react-router-dom";
 
+const linkStyle = {
+    textDecoration: "none",
+    fontSize: "small",
+}
+
 const Nav = () => {
     return (
         <nav className="Nav">
-            <Link to="/" >Home</Link>
-            <Link >Users</Link>
-            <Link >Coding</Link>
-            <Link >Football</Link>
-            <Link >Cooking</Link>
+            <Link to="/" style={linkStyle}>HOME</Link>
```

Comment:
Again, these changes are not related to your current ticket. 

You will also find that displaying the topics this way is quite limiting and not at all future-proof. 

Could you make use of a different endpoint on your api to fetch the topics...?